packages/frontend/src/layouts/AuthenticatedLayout.tsx

```diff
@@ -4,17 +4,27 @@ import { Dialog, Transition } from '@headlessui/react';
 import { faX, faBars } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useRecoilValue } from 'recoil';
-import { SingleSetting, useAllSettingsQuery } from '@/model/settings';
+import {
+  SingleSetting,
+  useAllSettingsQuery,
+  useGithubVersionQuery,
+} from '@/model/settings';
 import { useAllPeriodsQuery } from '@/model/periods';
 import { useAllUsersQuery } from '@/model/users';
-import { ActiveUserRoles } from '@/model/auth';
+import { ActiveUserRoles, HasRole, ROLE_ADMIN } from '@/model/auth';
 import { Nav } from '@/navigation/Nav';
 import { AuthenticatedRoutes } from '@/navigation/AuthenticatedRoutes';
 
 export const AuthenticatedLayout = (): JSX.Element | null => {
   const [sidebarOpen, setSidebarOpen] = useState(false);
   const siteNameSetting = useRecoilValue(SingleSetting('NAME'));
   const activeUserRoles = useRecoilValue(ActiveUserRoles);
+  const isAdmin = useRecoilValue(HasRole(ROLE_ADMIN));
+
+  const githubVersion = useGithubVersionQuery().substring(1);
```

Comment:
Consider moving the version handling logic to the hook as well to make the setup more clear and clean. Hook could then be named `usePraiseAppVersion` and return an object in the format:

```
{
  current: "v0.9.0",
  latest: "v0.10.0",
  newVersionAvailable: true
}
```